diff --git a/test_beds/sanity_checks.py b/test_beds/sanity_checks.py
index 5da10d6..e2adf66 100644
--- a/test_beds/sanity_checks.py
+++ b/test_beds/sanity_checks.py
@@ -82,7 +82,7 @@ parser.add_argument('--normalize_action', type=bool, default=False, metavar='NOR
                     help='ENV: normalize environment action space (default: False)')
 
 ## Add arguments for soft actor-critic algorithm
-parser.add_argument('--do_logging', type=bool, default=True, metavar='DO_LOG',
+parser.add_argument('--do_logging', type=bool, default=False, metavar='DO_LOG',
                     help='SAC_A: Activate training logging (default: True)')
 parser.add_argument('--algorithm', type=str, default='SAC', metavar='RL_ALG',
                     help='SAC_A: RL algorithm type for AST (default: "SAC")')
@@ -1458,7 +1458,7 @@ if test8:
 
 # Test ast_sac_rollout() alone
 test9=True
-test9=False
+# test9=False
 
 if test9:
     start_time = time.time()
